packages/ui/src/components/modal/NewModal.vue

```diff
@@ -60,8 +60,6 @@ const props = withDefaults(
     closeOnClickOutside?: boolean
     warnOnClose?: boolean
     header?: string
-    onHide?: () => void
```

Comment:
Emits is definitely a much better way to do this lol, though I think this probably breaks the things that use it which didn't seem to be updated? Unless :on-hide is valid for an emits subscription same as @on-hide?

Reply:
oh, I tried to use IntelliJ to see usages but it does not seem happy with the project, I'll look on the usages using find and update them accordingly if required

Reply:
seeing that all other usages are on the modrinth app, I went back on the change as I daily drive a Linux computer with an Nvidia GPU